### DISCUSSION TITLE
Help wanted for using jeager to trace blockchain ID.

### DISCUSSION THREAD

**Discussion by SamYuan1990** [jaegertracing/jaeger#3389] [Q&A]
Hi guys,

It is well known as blockchain with hash string there can be used as UUID for all network.
So I want to use jaeger to trace blockchain transaction with UUID.
I did some attempt as below
```golang
span = opentracing.GlobalTracer().StartSpan("business label", opentracing.Tag{Key: "txid", Value: Txid})
defer span.finished()
sth()
```

I can get the txid from ui as `a70d1b9632f6885f265e7ab58f323ea14c0d90f6adc1052e27a393fed47a543d `

and I want to trace the txid at ui here
![image](https://user-images.githubusercontent.com/7820992/141684611-5041d256-c008-4ea0-83a3-9cc7bf5507cb.png)

but I received:
![image](https://user-images.githubusercontent.com/7820992/141684637-eb5161b5-1389-4dc4-ac48-661ae850ebbe.png)

please advise.

thanks and regards
Sam

**Comment by SamYuan1990** [comment]
```
txid=31a230b23c67f854ec4e9505283270055e16259d0a07e210f27cb8decb64d086
```

ok, I find how to use it in screen shot of https://github.com/jaegertracing/jaeger/issues/2540

**Comment by pavolloffay** [comment]
The `Txid` can be queried as a tag not as a traceid.
